test/vars/MavenSpec.groovy

```diff
@@ -101,4 +101,14 @@ class MavenSpec extends JenkinsPipelineSpecification {
         then:
         1 * getPipelineMock("sh")('mvn -B -s settingsFileId -fae clean install')
     }
+
+    def "[maven.groovy] update pom element"() {
+        setup:
+            def pomURL = "https://raw.githubusercontent.com/kiegroup/kogito-runtimes/master/pom.xml"
+            def pom = new URL(pomURL).getText();
+        when:    
+            mavenGroovy.updatePomElement(pom,"version.maven","0.0.0")
+        then:
+            notThrown(MissingPropertyException)
+    }
 }
```

Comment:
The tests (two of them because the `if`) for `updatePomElement` is missing, please add it

Reply:
The "notThrown(MissingPropertyException)" covers the "No element found called $ {elementName}" part and the " mavenGroovy.updatePomElement(pom,"version.maven","0.0.0")" covers the "else".